src/main/scala/org/apache/spark/sql/redis/HashRedisPersistence.scala

```diff
@@ -39,6 +39,13 @@ class HashRedisPersistence extends RedisPersistence[Any] {
         // don't store key values
         k != keyName
       }
+      .filter { case (k, _) =>
+        // don't store TTLs
+        ttlColumn match {
```

Comment:
It's probably more efficient in terms of the performance to check if ttlColumn is specified only once rather than for every field.